nautobot/core/views/generic.py

```diff
@@ -313,7 +323,8 @@ def get(self, request):
             "permissions": permissions,
             "action_buttons": valid_actions,
             "table_config_form": TableConfigForm(table=table),
-            "filter_form": filter_form,
+            "filter_params": display_filter_params,
+            "filter_form": self.dynamic_filter_form,
```

Comment:
1. I believe that 'Filterset' naming standard should be mandated,
2. Due to the fundamental differences between FilterForm and "DynamicFilterForm," it would not function.

Reply:
I agree with you on 1. but the fact remains there may be existing plugins that are not following this convention, and we'd like to avoid breaking them if at all possible.

Reply:
Do you want support for both FilterForm and DynamicFilterForm?
If yes, I think we should create an issue, cause I taught, the objective was to drop support for FilterForm and use DynamicFilterForm instead.